src/Thenewboston/Bank/Api/BankBankService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Thenewboston.Bank.Api.Models;
+using Thenewboston.Bank.Models;
+using Thenewboston.Common.Api.Models;
+using Thenewboston.Common.Http;
+
+namespace Thenewboston.Bank.Api
+{
+    public class BankBankService : IBankBankService
```

Comment:
You have a fair concern regarding naming.
Here's my idea: let's just call it BanksService (or ConnectedBanksService, to be more clear). I'll handle renaming the other classes to follow the same pattern: AccountService, TransactionsService etc.
While this would lead to some classes from different namespaces sharing the same name (from validator and bank), this should not be a concern for users, as we will not expose these classes for them (I know they're public now, but we'll handle that). 

Let me know if that makes sense.